hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/StateContext.java

```diff
@@ -332,19 +363,21 @@ public void putBackReports(List<GeneratedMessage> reportsToPutBack,
     return reportsToReturn;
   }
 
-  List<GeneratedMessage> getNonIncrementalReports() {
+  List<GeneratedMessage> getNonIncrementalReports(
```

Comment:
Let's rename this to `getFullReports()` then.

Reply:
sure, of course!